### ISSUE TITLE
Import simple convex stl file

### ISSUE THREAD

**Issue by mkeilman** [radiasoft/sirepo#4829]
Build the basic import mechanism around a simple file such as 
[cube_1_mm.stl.txt](https://github.com/radiasoft/sirepo/files/9552099/cube_1_mm.stl.txt)

**Comment by mkeilman** [comment]
Quick ping @Benbenbenin0 - lmk when you have started on this and if you have any questions. Incidentally I suggest writing a test (see #4849) for this in parallel, since the GUI part is a bit more involved.

**Comment by Benbenbenin0** [comment]
> Quick ping @Benbenbenin0 - lmk when you have started on this and if you have any questions. Incidentally I suggest writing a test (see #4849) for this in parallel, since the GUI part is a bit more involved.

Besides making edits to a previous PR this is my current issue I'm working on. I've familiarized a bit with the GUI code and am looking into writing the convexity test.

**Comment by mkeilman** [comment]
> > Quick ping @Benbenbenin0 - lmk when you have started on this and if you have any questions. Incidentally I suggest writing a test (see #4849) for this in parallel, since the GUI part is a bit more involved.
> 
> Besides making edits to a previous PR this is my current issue I'm working on. I've familiarized a bit with the GUI code and am looking into writing the convexity test.

If you mean you are writing your own code to test for the convexity of a polyhedron, that should not be necessary. The trimesh library does that work, which our tests should invoke.

**Comment by Benbenbenin0** [comment]
Right I realized I'm just using the trimesh repo after making the comment. I also grabbed the image from the branch you made (thank you), I have a branch for issue #4848.

**Comment by Benbenbenin0** [comment]
Issue merged into #4848
